connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/RestServerTest.java

```diff
@@ -166,6 +166,7 @@ public void testOptionsDoesNotIncludeWadlOutput() throws IOException {
         Map<String, String> configMap = new HashMap<>(baseWorkerProps());
         DistributedConfig workerConfig = new DistributedConfig(configMap);
 
+        EasyMock.expect(herder.kafkaClusterId()).andReturn(null);
```

Comment:
Does it make sense to actually declare the kafka cluster id as in the previous test (member field) and return it here? Maybe the code might start using it at some point.

Reply:
Sure, couldn't hurt.